.github/workflows/build.yml

```diff
@@ -180,3 +180,64 @@ jobs:
           INGRESS_DOMAIN: radix-dapp-toolkit-dev.${{ secrets.INGRESS_DOMAIN }}
           HELM_GH_USER: ${{ secrets.HELM_GH_USER }}
           HELM_GH_PASS: ${{ secrets.HELM_GH_PASS }}
+
+  deploy-release:
+    if: contains('release', github.ref_name)
```

Comment:
Wondering if we can use pattern matching like `release/*` , if it is easy to change?

contains('release', github.ref_name) matches branches with release anywhere and so say someone fixing issue for release might create a branch like `fix/update_something_for_ash_release`